python/test_python.py

```diff
@@ -39,6 +40,43 @@ def docker_container(request):
     container.stop()
     container.remove()
 
+
+@pytest.fixture
+def docker_client():
+    return docker.from_env()
```

Comment:
Is there a scenario in which we would use a different docker client instead of this fixture? Curious as to why you don't just hard code the docker client into the `docker_image` fixture below?

Reply:
> Is there a scenario in which we would use a different docker client instead of this fixture?
There might be a scenario in the future, but until then, yeah, it should be hard coded in the `docker_image` fixture. Good catch.

Reply:
Actually, when I went to hardcode the `docker_client` fixture inside the `docker_image` fixture, I realized that the `docker_runner` needs both the `docker_client` and `docker_image` fixture, so we can't remove the `docker_client` fixture without duplicating the code.